src/quick-tour/moment-of-purchase.md

```diff
@@ -4,29 +4,26 @@ title: Moment of Purchase
 
-[Order Processing]({{ site.baseurl }}{% link sales/order-processing.md %})
-:  Magento supports a complete order processing workflow. It's easy to customize order statuses and track communications between sales reps and customers.
+Order Processing
+:  Magento supports a complete [order processing]({{ site.baseurl }}{% link sales/order-processing.md %}) workflow. It is easy to customize order statuses and track communications between sales reps and customers.
 
-[Multiple Payment Options]({{ site.baseurl }}{% link payment/payments.md %})
-:  Magento supports the payment methods and currencies needed for global commerce. You can choose the ones you want to offer, and at checkout, your customers can choose the ones they prefer.
+Multiple Payment Options
+:  Magento supports the [payment methods]({{ site.baseurl }}{% link payment/payments.md %}) and currencies needed for global commerce. You can choose the ones you want to offer, and your customers can choose the ones they prefer at checkout.
 
-[PayPal Merchant Solutions]({{ site.baseurl }}{% link payment/paypal.md %})
-:  It's easy to integrate a PayPal Payments account to provide your customers faster, more secure checkout options.
+PayPal Merchant Solutions
+:  It is easy to integrate a [PayPal Payments]({{ site.baseurl }}{% link payment/paypal.md %}) account to provide your customers faster, more secure checkout options.
 
-[Magento Shipping]({{ site.baseurl }}{% link shipping/magento-shipping.md %})
-:  Offer seamless access to global carrier networks, and use customer experience rules to streamline operations and automate processes.
+Magento Shipping
+:  [Magento Shipping]({{ site.baseurl }}{% link shipping/magento-shipping.md %}) provides seamless access to global carrier networks, and you can use customer experience rules to streamline operations and automate processes. It supports a variety of shipping methods so you can give your customers a choice at checkout. Customers can see a real-time estimate of shipping charges right from the shopping cart.
 
-[Multiple Shipping Options]({{ site.baseurl }}{% link shipping/shipping.md %})
-:  Magento supports a variety of shipping methods so you can give your customers a choice at checkout. Customers can see a real-time estimate of shipping charges right from the shopping cart.
-
-[Shipping Labels]({{ site.baseurl }}{% link shipping/shipping-labels.md %})
-:  Merchants have complete control over package characteristics such as weight and size. Shipping labels, rate, and bar code information originates directly from the carrier. Labels can be generated for single or multiple orders.
+Shipping Labels
+:  Merchants have complete control over package characteristics such as weight and size. [Shipping labels]({{ site.baseurl }}{% link shipping/shipping-labels.md %}), rate, and bar code information originates directly from the carrier. Labels can be generated for single or multiple orders.
```

Comment:
might be too picky.  the last sentence.   should that be single- or multiple-item orders?

Reply:
rewrote... that isn't what it means but I made it clearer.